src/css/styles/_layout.scss

```diff
@@ -51,3 +51,21 @@
     padding: 0;
   }
 }
+
+.l-form-container {
+  position: fixed;
+  top: 0;
+  left: 100%;
+  z-index: 1010;
+  width: 100%;
+  height: 100%;
+  overflow-y: scroll;
+  background: $col-bg-gray-dark;
+  opacity: 0;
+  transition: 0.5s ease-in-out;
```

Comment:
I don't like that `transition` effect

- [ ] remove?
- [ ] fade-in? (https://stackoverflow.com/a/39669911/2051256)